.dockerignore

```diff
@@ -0,0 +1,26 @@
+.DS_Store
+.git
+.svn
```

Comment:
Wouldn't be a bad idea, was mainly trying to avoid large directories in the copy.